lib/krikri/enrichments/split_coordinates.rb

```diff
@@ -0,0 +1,73 @@
+module Krikri::Enrichments
+  ##
+  # Splits a string given in `lat' or `long' in an edm:Place object and
+  # assigns `lat' and `long' with the split values.
+  #
+  # @example
+  #
+  #   Where val is a DPLA::MAP::Place,
+  #   and val.lat == '40.7127,74.0059',
+  #   assign val.lat = '40.7127' and val.long = '74.0059'
+  #
+  #   If long is filled in instead of lat, the values will be assigned in the
+  #   reverse order, with lat taking '74.0059' and long taking '40.7127'.
+  #
+  class SplitCoordinates
+    include Krikri::FieldEnrichment
+
+    ##
+    # Enrich a `DPLA::MAP::Place' object by splitting the string given
+    # in its `lat' or `long'.
+    #
+    # @param place [DPLA::MAP::Place]
+    # @return [DPLA::MAP::Place]
+    #
+    def enrich_value(place)
+      return place if !place.is_a? DPLA::MAP::Place
+
+      # place.lat and place.long are arrays of one value if they are assigned.
+
+      return place if place.lat.first && place.long.first  # no split required
```

Comment:
This requires that the mapping be to one of `lat` or `long`, but not both.  That's not what we're doing to date.  Would it be better to check whether either value contains a comma?